shared/test/dataDriver_test.js

```diff
@@ -298,6 +298,17 @@ describe("loop.DataDriver", () => {
     });
   });
 
+  describe("#requestRoomData", () => {
+    it("should request everything but chat", () => {
+      driver.requestRoomData();
+
+      let queries = requests[0].url.split("?")[1].split("&").sort();
+      expect(queries).contain("orderBy=%22%24key%22");
+      expect(queries).contain("startAt=%22chat~%22");
+      expect(queries).contain(`limitToLast=${driver.MAX_LIMIT}`);
```

Comment:
This wants a few helpers, probably on dataDriver itself, to make this more readable.  That said, feel free to just add an XXX here so that someone considers doing it in the future.